.env.example

```diff
@@ -1,2 +1,3 @@
 GA_TRACKING_ID=your-ga-tracking-id
 SENTRY_JS_DSN=https://foo123@sentry.service.dsd.io/99
+PROMETHEUS_METRICS="string"
```

Comment:
I remember doing this in OMIC but wonder if it makes sense to change to KUBERNETES_DEPLOYMENT=boolean to be more explicit about why the Prometheus Exporter exists?

Reply:
You will need to add the env var to your deployment.yaml if you have not done so already

Reply:
Env var has been added to deployment.yaml, which is another reason I know that I'd previously written up these files, but they somehow got lost.

Happy to switch to true/false if you think that's a better approach